drongr/drongr.css

```diff
@@ -0,0 +1,53 @@
+.exhibit{
+    width:1024px;
+    text-align: center;
+    font-family: 'Times New Roman', Times, serif;
+    margin: auto;
+    color: #9b9b9b;
+}
+.illustrate{
+    width:450px;
```

Comment:
You also need add a whitespace(` `) between the css property (eg: `width`) and the css value (eg: `1024px`).
eg:
```css
width: 1024px;
```